dashboards/Home_Dashboard.json

```diff
@@ -149,39 +147,56 @@
             "datasource": null,
             "gridPos": {
                 "h": 11,
-                "w": 12,
+                "w": 7,
                 "x": 0,
                 "y": 7
             },
             "height": "330px",
             "id": 1021,
             "links": [],
             "mode": "markdown",
-            "options": {},
             "title": "",
             "type": "text"
         },
         {
             "datasource": null,
+            "description": "",
+            "gridPos": {
+                "h": 11,
+                "w": 7,
+                "x": 7,
+                "y": 7
+            },
+            "id": 1396,
+            "options": {
+                "feedUrl": "https://cors-anywhere.herokuapp.com/https://www.percona.com/blog/feed"
```

Comment:
We should point this to pull the feed directly from `https://www.percona.com/blog/feed/`.
To overcome cors, we should add the cors headers to `https://github.com/percona/pmm-server/blob/PMM-2.0/nginx.conf`